addons/mail/models/res_partner.py

```diff
@@ -318,8 +317,10 @@ def im_search(self, name, limit=20, excluded_ids=None):
         return list(users.partner_id.mail_partner_format().values())
 
     @api.model
-    def _get_current_persona(self):
-        guest = self.env["mail.guest"]._get_guest_from_context()
-        if self.env.user._is_public() and guest:
+    def _get_current_persona(self, raise_on_missing=True):
```

Comment:
`raise_on_missing_guest`?
This is a weird API shouldn't it be the caller job to raise if there is no guests?

Reply:
Agree, I'll remove it. Almost all calls are done with False anyway.